services/distribution/src/test/resources/master-index-test/expected_result.txt

```diff
@@ -0,0 +1,31 @@
+date/2020-05-11/hour/17
```

Comment:
All the static test files can be removed when index generation is done as decorator. Then the test could look similar to [this](https://github.com/corona-warn-app/cwa-server/blob/master/services/distribution/src/test/java/app/coronawarn/server/services/distribution/assembly/structure/directory/decorators/indexing/IndexingDecoratorTest.java#L71).